cmd/ak/cmd/runtimes/runtime.go

```diff
@@ -37,7 +37,8 @@ func init() {
 
 func runtimes() sdkservices.Runtimes {
 	if local {
-		return backendRuntimes.New()
+		// TODO: what is local ? what to do about logger ? which config ?
```

Comment:
Good questions :)
@itayd?

Reply:
not sure i understand the question here. `zap.New(nil)` looks very odd. i think we should do https://github.com/autokitteh/autokitteh/pull/663, but i can redo after merge.

Reply:
I just don't understand what's the purpose of local, what does it mean ?
if #663  solves it, lets do that after we merge this